Sources/Combine/Extensions/UIControl+Combine.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  UIControl+Combine.swift
+//  Catalog
+//
+//  Created by Zvonimir Medak on 29.10.2021..
+//  Copyright © 2021 Infinum. All rights reserved.
+//
+
+import UIKit
+import Combine
+
+// https://github.com/CombineCommunity/CombineCocoa/blob/main/Sources/CombineCocoa/CombineControlProperty.swift
```

Comment:
Since we're importing community libs in N&B, I don't see an issue with just importing CombineCocoa as a pod as well since we could maybe use something more from it down the road 🙂